### PR TITLE
Use pre-generated initdb.tar.zst in test_ingest_real_wal

### PR DESCRIPTION
**Pull request by arpad-m**
This implements the TODO mentioned in the test added by #5892

### CODE REVIEW

**Comment by arpad-m** [comment]
slack thread: https://neondb.slack.com/archives/C033RQ5SPDH/p1701885004596269

File: pageserver/src/walingest.rs [hunk 1]
```diff
@@ -2177,21 +2178,30 @@ mod tests {
...
         let startpoint = Lsn::from_hex("14AEC08").unwrap();
         let endpoint = Lsn::from_hex("1FFFF98").unwrap();
 
+        let harness = TenantHarness::create("test_ingest_real_wal").unwrap();
+        let (tenant, ctx) = harness.load().await;
+        let tenant_shard_id = tenant.tenant_shard_id();
+
+        // This instrumentation is needed to bootstrap the timeline
+        let span = tracing::info_span!(parent: None, "test_ingest_real_wal", tenant_id=%tenant_shard_id.tenant_id, shard_id=%tenant_shard_id.shard_slug(), timeline_id=%TIMELINE_ID);
+        let _enter = span.enter();
```
Comment (koivunej) [inline pageserver/src/walingest.rs:2191]: I was initially going to comment about use of `Span::enter` in async context but then saw that the added code is blocking...

Why add the span? it cannot be any other test running on these lines?

Comment (arpad-m) [reply pageserver/src/walingest.rs]: I now use `tracing::instrument`

**Review by koivunej — APPROVED** [review]

File: pageserver/src/walingest.rs (hunk 1 above)
Comment (koivunej) [reply pageserver/src/walingest.rs]: Looking good.

**Pull request merged**